pex/pep425tags.py

```diff
@@ -271,10 +271,16 @@ def get_supported(versions=None, noarch=False, platform=None,
         abis[0:0] = [abi]
 
     abi3s = set()
-    import imp
-    for suffix in imp.get_suffixes():
-        if suffix[0].startswith('.abi'):
-            abi3s.add(suffix[0].split('.', 2)[1])
+    if sys.version_info[0] == 2:
```

Comment:
Please see the header on this file - it's complicated:
https://github.com/pantsbuild/pex/blob/5dfc8c2bb717ead39a897c82320d9d9e4c8bdb84/pex/pep425tags.py#L1-L15

At the very least, the modified code here should be marked.

Better might be to update our vendored version of setuptools and implement #690.
That, though won't work since master of setuptools still uses `imp` unconditionally: https://github.com/pypa/setuptools/blob/5b128e8c82f4886b4ff110219de80a61bf8aa3b0/setuptools/pep425tags.py#L257-L260.
That said, pip, which it copies from, does do a conditional import now: https://github.com/pypa/pip/blob/6f0e873182e45d2f7a4b61b9800599723d2636f1/src/pip/_internal/utils/compat.py#L205-L214
Unfortunately though, pip's pep425tags.py is no longer self-contained.